services/user/Branding/ui/src/App.tsx

```diff
@@ -20,7 +20,7 @@ export const App = () => {
     const init = async () => {
         await supervisor.onLoaded();
         supervisor.preLoadPlugins([{ service: "branding" }]);
-        await getNetworkName();
+        setTimeout(getNetworkName, 1500);
```

Comment:
Does awaiting not work?

Reply:
@cool-ant

Reply:
Good catch; I did that for the demo to emulate what won't work now because... ?Supervisor?
So no, it wasn't working.
I'll create a PR to reverse this change.

Reply:
Nothing to do with Supervisor. I just couldn't verify this yet without post-install steps for chainmail